### PR TITLE
Update info about basic-auth support

### PR DESCRIPTION
**Pull request by DnCDevelopment**

### CODE REVIEW

File: source/developer/slash-commands.rst [hunk 1]
```diff
@@ -72,6 +72,9 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
   .. note::
     `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. Enable them from **System Console > Integrations > Custom Integrations** in prior versions or **System Console > Integrations > Integration Management** in versions after 5.12 or ask your System Administrator.
 
+  .. note::
+    `Custom commands urls do not support basic auth. Commands provide their own authentication level, thats why no need in basic auth support
```
Comment (justinegeffen) [inline source/developer/slash-commands.rst]:
```suggestion
    `Custom command URLs do not support basic authentication as the slash commands provide their own authentication level. 
```

File: source/developer/slash-commands.rst [hunk 2]
```diff
@@ -72,6 +72,9 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
   .. note::
     `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. Enable them from **System Console > Integrations > Custom Integrations** in prior versions or **System Console > Integrations > Integration Management** in versions after 5.12 or ask your System Administrator.
```
Comment (justinegeffen) [inline source/developer/slash-commands.rst]:
```suggestion
    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. Enable them from **System Console > Integrations > Integration Management** (**System Console > Integrations > Custom Integrations** in versions prior to 5.12) or ask your System Administrator.
```

File: source/developer/slash-commands.rst (hunk 1 above)
Comment (amyblais) [reply source/developer/slash-commands.rst]: Agree,

**Comment by amyblais** [comment]
@DnCDevelopment Would you be open to making the suggested changes?

**Comment by jasonblais** [comment]
Hey @DnCDevelopment, it's been a few weeks since we've heard from you, just touching base to hear if you have any questions about the feedback above?

**Comment by justinegeffen** [comment]
Hi @DnCDevelopment. I wanted to follow up on this and find out whether you'd be open to making the suggested changes. Let us know if anything is unclear. :)

File: source/developer/slash-commands.rst [hunk 3]
```diff
@@ -70,8 +70,11 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
...
+    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. Enable them from **System Console > Integrations > Integration Management** (**System Console > Integrations > Custom Integrations** in versions prior to 5.12) or ask your System Administrator.
 
+  .. note::
+    `Custom command URLs do not support basic authentication as the slash commands provide their own authentication level.
```
Comment (justinegeffen) [inline source/developer/slash-commands.rst]:
```suggestion
    Custom command URLs do not support basic authentication as the slash commands provide their own authentication level. 
```

**Comment by justinegeffen** [comment]
Hi @DnCDevelopment - I've committed the suggested formatting changes and wanted to find out whether you'd be open to reviewing the PR again to ensure the changes are aligned. Thank you!

File: source/developer/slash-commands.rst [hunk 4]
```diff
@@ -70,8 +70,11 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
...
 
   .. note::
-    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. Enable them from **System Console > Integrations > Custom Integrations** in prior versions or **System Console > Integrations > Integration Management** in versions after 5.12 or ask your System Administrator.
+    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. Enable them from **System Console > Integrations > Integration Management** (**System Console > Integrations > Custom Integrations** in versions prior to 5.12) or ask your System Administrator.
```
Comment (justinegeffen) [inline source/developer/slash-commands.rst:73]:
```suggestion
    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. Enable them from **System Console > Integrations > Custom Integrations** (or **System Console > Integrations > Integration Management** in versions prior to 5.12) or ask your Mattermost System Admin.
```

File: source/developer/slash-commands.rst [hunk 5]
```diff
@@ -70,8 +70,12 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
...
+    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to ``true`` in ``config.json`` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. 
+    Enable them from **System Console > Integrations > Custom Integrations** (or **System Console > Integrations > Integration Management** in versions prior to 5.12) or ask your Mattermost System Admin.
 
+  .. note::
+    Custom command URLs do not support basic authentication as the slash commands provide their own authentication level.
```
Comment (justinegeffen) [inline source/developer/slash-commands.rst:77]:
```suggestion
    Custom command URLs do not support basic authentication as the slash commands provide their own authentication level.
```

File: source/developer/slash-commands.rst [hunk 6]
```diff
@@ -70,8 +70,12 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
 5 - (Optional) Set the response username and icon the command will post messages as in Mattermost. If not set, the command will use your username and profile picture.
 
   .. note::
-    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__. Enable them from **System Console > Integrations > Custom Integrations** in prior versions or **System Console > Integrations > Integration Management** in versions after 5.12 or ask your System Administrator.
+    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to ``true`` in ``config.json`` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__.
```
Comment (justinegeffen) [inline source/developer/slash-commands.rst:73]:
```suggestion
    `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to ``true`` in ``config.json`` to override usernames, and `similarly for profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__.
```

**Comment by justinegeffen** [comment]
Hi @DnCDevelopment. :) I've committed some stylistic changes and wanted to confirm the content with you so we can merge the PR and have your changes published. :)

**Review by aaronrothschild — APPROVED** [review]
LGTM, sorry for extremely late review - cleaning up some old tickets and this was obscured.

**Comment by justinegeffen** [comment]
The forked repo has been deleted and this can't be merged.

**Pull request closed without merging**